astropy/modeling/core.py

```diff
@@ -668,11 +668,6 @@ class Model(metaclass=_ModelMeta):
     in the class body.
     """
 
-    n_inputs = 0
```

Comment:
This would need inputs from @nden or @WilliamJamieson . Surely they were done as such for a reason, maybe to ensure it returns 0 in some cases?

Reply:
Actually this is directly breaking the properties with the same names, who rely on the class attribute as a default. Sorry, I should have read this more carefully. I think I can simply give the class attributes a different (private) name to satisfy the linter.

Reply:
@nden , are you okay with this?

Reply:
@WilliamJamieson ?

Reply:
Maybe I should point out that those attributes were already not accessible at runtime as they are shadowed by properties with the same names (which is what ruff is pointing out).

Reply:
I should also link #16633 were I did a more thorough cleanup

Reply:
Given https://github.com/astropy/astropy/pull/16633 , should we just not touch modeling here?

Reply:
Well I have to do something about it here to pass linting, and this bit is the same so there should not be any conflict.